backvendor/gosource.go

```diff
@@ -217,9 +353,9 @@ func importPathFromFilepath(pth string) (string, bool) {
 		}
 
 		pth := strings.Join(components[i:len(components)], "/")
```

Comment:
Could you please use a different name? `pth` is a function argument and it makes the code below confusing if you didn't read this line.

Reply:
Changed.